src/components/Navigation/ConversationsList/ConversationsList.vue

```diff
@@ -45,20 +45,30 @@ export default {
 			return this.$store.getters.conversations
 		}
 	},
-	async beforeMount() {
-		/** Fetches the conversations from the server and then
-		 * adds them one by one to the store.
-		 */
-		const conversations = await fetchConversations()
-		conversations.data.ocs.data.forEach(conversation => {
-			this.$store.dispatch('addConversation', conversation)
-		})
+	beforeMount() {
+		this.fetchConversations()
```

Comment:
What would be best is to have an initialstate for this!
Would be much more snappier! ;)

Reply:
What do you mean??

Reply:
Something for later